modules/accredited_representative_portal/app/serializers/accredited_representative_portal/application_serializer.rb

```diff
@@ -19,7 +19,7 @@ def serializable_hash
     private
 
     def unwrap_serializable_hash(data)
-      data[:attributes].merge!(id: data[:id])
+      data[:attributes].merge!(id: data[:id]) unless data[:id].nil?
```

Comment:
@chumpy The return value for this method is the result of serialization. `nil` `:id` is a valid case. This modification will make the return value `nil` when `:id` isn't present, which seems wrong.